### PR TITLE
OCM-7315 | feat: describe client support for configurable registries

### PR DESCRIPTION
**Pull request by chenz4027**
https://issues.redhat.com/browse/OCM-7315

### CODE REVIEW

File: cmd/describe/cluster/cmd.go [hunk 1]
```diff
@@ -837,6 +842,47 @@ func getAuditLogForwardingStatus(cluster *cmv1.Cluster) string {
 	return auditLogForwardingStatus
 }
 
+func getClusterRegistryConfig(cluster *cmv1.Cluster) string {
+	var output string
+	if cluster.RegistryConfig().RegistrySources() != nil {
+		registryResources := cluster.RegistryConfig().RegistrySources()
```
Comment (andreadecorte) [inline cmd/describe/cluster/cmd.go:848]: let's get them in a dedicated section like Network

Comment (chenz4027) [reply cmd/describe/cluster/cmd.go:849]: 👍 Added https://github.com/openshift/rosa/pull/2472/files#diff-15a63c5c54a2463b4203ef656bf435bf0e1a4f9725eecbb2015d503d0dc298b9R501

**Review by mikeshng — APPROVED** [review]
/lgtm

**Pull request merged**